pkgs/development/python-modules/asyncpg/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, isPy3k, fetchPypi, buildPythonPackage
+, flake8, uvloop, postgresql }:
+
+buildPythonPackage rec {
+  pname = "asyncpg";
+  version = "0.18.3";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0rrch478ww6ipmh3617sb2jzwsq4w7pjcck869p35zb0mk5fr9aq";
+  };
+
+  # A useless patch, since flake8 tests are going to fail anyway
+  patches = [
+    ./include-dot-flake8.patch
```

Comment:
Why did you include it, then? Just get rid of all comments regarding flake8, no need to test for code quality in nixpkgs.

Reply:
Oh, the patch is actually needed for the tests to just not fail, since it first checks if `.flake8` exists before checking whether or not `flake8` is there and skipping if it isn't.